spec/features/user_can_retwit_twit_spec.rb

```diff
@@ -42,4 +42,29 @@
       end
     end
   end
+
+  scenario "User tries to reTwits their own Twit" do
+    VCR.use_cassette("user_retwits_their_own_twit") do
+      visit root_path
+      click_link "Login"
+      within("form[action='/twits']") do
+        fill_in "twit[message]", with: "This is a test of the Twitter API system."
+        find("button[type='submit']").click
+      end
+
+      new_twit_footer =
+        find(".panel-body",
+             text: "This is a test of the Twitter API system.")
+        .first(:xpath,".//..")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Space missing after comma.